src/components/Application-Header/JobsAppBar.tsx

```diff
@@ -4,12 +4,12 @@ import { withStyles, WithStyles } from '@material-ui/core';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
-import AccessibilityNewIcon from '@material-ui/icons/AccessibilityNew';
 import Avatar from '@material-ui/core/Avatar';
 import IconButton from '@material-ui/core/IconButton';
 import StarIcon from '@material-ui/icons/Star';
 import Tooltip from '@material-ui/core/Tooltip';
 
```

Comment:
remove line

Reply:
that line is there for logical separation of the imports, 
actually @JetoPistola  asked me add that line.